server/channels/store/storetest/user_store.go

```diff
@@ -4488,6 +4489,39 @@ func testUserStoreAnalyticsGetInactiveUsersCount(t *testing.T, rctx request.CTX,
 	require.Equal(t, count, newCount-1, "Expected 1 more inactive users but found otherwise.")
 }
 
+func testUserStoreAnalyticsGetInactiveUsersCountIgnoreBots(t *testing.T, rctx request.CTX, ss store.Store) {
+	u1 := &model.User{}
+	u1.Email = MakeEmail()
+	_, err := ss.User().Save(rctx, u1)
+	require.NoError(t, err)
+	defer func() { require.NoError(t, ss.User().PermanentDelete(rctx, u1.Id)) }()
+
+	count, err := ss.User().AnalyticsGetInactiveUsersCount()
```

Comment:
Should we check the value of `count`?

Reply:
I would expect any problems with count to be detected by `AnalyticsGetInactiveUsersCount`

anything specific you wanted to test?

Reply:
that's right, I've added it